src/components/common/Actions.vue

```diff
@@ -208,6 +211,10 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Name
   private async onClickFilePay (): Promise<void> {
     // Prompt Step validations
     this.setValidateSteps(true)
+    if (this.isTypeFirm) {
```

Comment:
This check should be in src/views/DissolutionFirm/DissolutionFirm.vue, not here.

You can see that a global flah (validateSteps) is set to True above. You can watch that flag (or just use reactivity) to display the error there.

This change will probably simply a lot of things in this PR.

Reply:
You found an even better solution 👍